requirements.txt

```diff
@@ -1,4 +1,5 @@
 numpy
+bottleneck==1.2.1
```

Comment:
Can be separate task but pinning requirements is good practice.

Reply:
Yes, I will pin requirements once I check the other PR works properly.